docs/community/index.md

```diff
@@ -17,5 +17,4 @@ If you would like to get involved with Aurae development.
 
 [Aurae](https://github.com/aurae-runtime/aurae) is an opinionated Turing complete scripting language built for the enterprise. Think of it like TypeScript for infrastructure platforms.
 
-[Auraed](https://github.com/aurae-runtime/auraed) is the project core runtime daemon, auth and identity management system, and gRPC server that listens on a Unix domain socket.
-
+[Auraed](https://aurae.io/auraed/) is the project core runtime daemon, auth and identity management system, and gRPC server that listens on a Unix domain socket.
```

Comment:
This link is broken. I chose to point it to the corresponding section on aurae.io instead of the GitHub repo path (https://github.com/aurae-runtime/auraed) because this link appears on an aurae.io webpage.